.eslintrc

```diff
@@ -1,39 +1,9 @@
+---
 parser: "babel-eslint"
 
-env:
-
-    ###########################################################################
-    #                                                                         #
-    #   ENVIRONMENT: if you write code that will be executed in one of the    #
-    #   following environments, the value for that environment should be      #
-    #   set to true.                                                          #
-    #                                                                         #
-    ###########################################################################
-
-    browser: true
-    node: true
-    amd: false
-    mocha: true
-    jasmine: false
-
-globals:
-
-    ###########################################################################
-    #                                                                         #
-    #   GLOBALS: ESLint will assume the following variables are declared      #
-    #   globally; other variables will require explicit declaration.          #
-    #                                                                         #
-    ###########################################################################
-
-    require: true
-    describe: true
-    beforeEach: true
-    afterEach: true
-    before: true
-    after: true
-    it: true
-    sinon: true
-    expect: true
+extends:
+  - "defaults"
+  - "defaults/rules/eslint/node/walmart"
```

Comment:
You were pretty close to the WM config so I aligned that section to override it but I can pull in the "off.js" file instead and let you 100% override or whatever you want to do. I just wanted to use this PR to get the ball rolling.